### DISCUSSION TITLE
Error: Your request was rejected because you have performed too many operations recently. You have been signed out, please sign in and try again later.

### DISCUSSION THREAD

**Discussion by KeikoMei** [WeblateOrg/weblate#6118] [Q&A]
I have been hired to translate a project on this platform. I've had some issues that I've never had before with a CAT tool. But this one doesn't let me work. 

Last week some folders were locked because of an error updating the files. As a result, some strings were not propagating to those folders. I thought that once those folders were unlocked it would be easy enough to propagate.

Today the issue was fixed and those folders are not locked anymore. So I start propagating the strings by using the filter "inconsistent". Since there is no way to do every string at once, I start clicking "save" on each, and this propagates the strings. But there is a point where I start to do it more quickly. An the tool decides that I cannot work quickly and logs me out, with the following message:

"Your request was rejected because you have performed too many operations recently. You have been signed out, please sign in and try again later. "

I don't know when I can try later. How long do I have to wait? 
Why coding a limit on "too many operations?
Do you realize that people are working with deadlines?
I can log in again but as soon as I try to save a change, the same thing happens.

**Comment by nijel** [comment]
> Since there is no way to do every string at once.

Automatic translation can do that in bulk.

>  Why coding a limit on "too many operations?

The default limit for translating is 30 strings in 60 seconds, see https://docs.weblate.org/en/latest/admin/optionals.html#rate-limiting
